### DISCUSSION TITLE
开设聊天区

### DISCUSSION THREAD

**Discussion by sld272** [Diaoxiaozhang/Ximalaya-Downloader#63] [讨论区]
大家有关于这个项目的一些问题、建议都可以发在这里。

**Comment by maple0308** [comment]
喜马拉雅免费的音频文件，一键下载之后，无法播放，用解密的版本解密XM格式的文件出现错误，应该是喜马拉雅官方调整了免费专辑音频的加密方式，希望大佬看到后可以更新一下。
